schema/deploy/shipping/views.sql

```diff
@@ -3528,7 +3465,7 @@ create or replace view shipping.__uw_priority_queue_v1 as (
         from uw_enrollments
         -- Filter to participants who enrolled after this date when
         -- baseline test invitations were handled manually
-        where encountered > '2020-12-15'
+        where encountered > '2021-11-15'
```

Comment:
This change is what is filtering out all of the rows from this view on my local db. If I change it back to the original date, I get 5905 rows but with the new date I get none (local database refreshed yesterday - 11/16).

Reply:
This date filter is just limiting the baseline testing invitations, correct? I don't think its expected to limit the other categories - and doesn't seem to in my testing. When I run the priority queue view, I see participants in the queue for  exposure_to_known_positive, symptomatic, and travel testing.